apps/src/code-studio/districtDropdown/SchoolInfoBox.jsx

```diff
@@ -0,0 +1,145 @@
+import React from 'react';
+// import i18n from '@cdo/locale'; // TODO
+
+function SchoolCountry(props) {
+  return (
+    <div className="itemblock">
+      <div className="labelblock">School Country</div>
+      <select className="form-control fieldblock" id="school-country" name="user[school_info_attributes][country]" type="select" value={props.country} onChange={props.onChange}>
+        <option value="US">United States</option>
+        <option value="AD">Andorra</option>
+        <option value="AE">United Arab Emirates</option>
+      </select>
+    </div>
+  );
+}
+
+SchoolCountry.propTypes = {
+  country: React.PropTypes.string,
+  onChange: React.PropTypes.func
+};
```

Comment:
Generally we aim for one component per file. These are small enough that I don't hate them sharing a file if that's your preference, but I'd recommending at least commenting each one individually.